evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -169,9 +169,8 @@ def test_error_metrics():
                                    pd.Series([-np.log(0.9), -np.log(0.2)]))
 
 
-input_features_and_y_true = [([1], None, "^Input features must be a dataframe with more than 10 rows!"),
-                             (pd.DataFrame({"a": [1]}), None, "^Input features must be a dataframe with more than 10 rows!"),
-                             (pd.DataFrame({"a": range(15)}), [1], "^Parameter y_true must be a pd.Series."),
+input_features_and_y_true = [([[1]], pd.Series([1]), "^Input features must be a dataframe with more than 10 rows!"),
+                             (pd.DataFrame({"a": [1]}), pd.Series([1]), "^Input features must be a dataframe with more than 10 rows!"),
```

Comment:
Having invalid parameters will now raise an error in the Woodwork conversion code instead, so will never reach these. I think this is okay, given our docstrings explicitly ask for pd.Series/ww.DataColumn.